rewriter/Minitest.cc

```diff
@@ -653,6 +654,99 @@ ast::ExpressionPtr runSingle(core::MutableContext ctx, bool isClass, ast::Send *
             return constantMover.addConstantsToExpression(send->loc, move(method));
         }
 
+        case core::Names::its().rawId(): {
+            if (block == nullptr || !send->recv.isSelfReference() ||
+                (!insideDescribe && requiresSecondFactor(send->fun))) {
```

Comment:
Unlike in some of the other cases (e.g., the describe/example group case, and the it/example case), there is only one name which triggers this logic: `its`.

So there's no need to indirect through `requiresSecondFactor`. `send->fun` is always `its`, and `requiresSecondFactor(its)` is always true.

```suggestion
                !insideDescribe) {
```